c10/cuda/CUDAStream.cpp

```diff
@@ -100,6 +104,7 @@ enum class StreamIdType : uint8_t {
   DEFAULT = 0x0,
   LOW = 0x1,
   HIGH = 0x2,
+  EXT = 0x4,
```

Comment:
The bit numbering here is a misunderstanding of what we need. This is not a bitmask. You can set EXT as 0x3.